bot/exts/moderation/infraction/_utils.py

```diff
@@ -170,8 +172,10 @@ async def notify_infraction(
     )
 
     # For case when other fields than reason is too long and this reach limit, then force-shorten string
-    if len(text) > 4096:
-        text = f"{text[:4093]}..."
+    if len(text) > 4096 - LONGEST_EXTRAS:
+        text = f"{text[:4093-LONGEST_EXTRAS]}..."
+
+    text += INFRACTION_APPEAL_SERVER_FOOTER if infr_type.lower() == 'ban' else INFRACTION_APPEAL_MODMAIL_FOOTER
```

Comment:
Is there a reason this is now being sent in the description rather than footer?

Reply:
This is to allow hyperlinking the guild invite